src/app/ror.org/[rorid]/Content.tsx

```diff
@@ -1,37 +1,38 @@
+'use client'
 import React from 'react'
 import Container from 'react-bootstrap/Container'
 import Row from 'react-bootstrap/Row'
 import Col from 'react-bootstrap/Col'
 
-import { Organization as OrganizationType } from 'src/data/types'
-import { fetchOrganization } from 'src/data/queries/organizationQuery';
+import type { MinimalOrganization as OrganizationType } from 'src/data/queries/searchOrganizationQuery'
+import { useROROrganization } from 'src/data/queries/searchOrganizationQuery';
 
 import Error from 'src/components/Error/Error'
 import Title from 'src/components/Title/Title'
 import DownloadReports from 'src/components/DownloadReports/DownloadReports'
 import ShareLinks from 'src/components/ShareLinks/ShareLinks'
-import Organization from 'src/components/Organization/Organization'
+import OrganizationMetadata from 'src/components/OrganizationMetadata/OrganizationMetadata'
+import SummarySearchMetrics from 'src/components/SummarySearchMetrics/SummarySearchMetrics'
+import Loading from 'src/components/Loading/Loading'
 import { rorFromUrl } from 'src/utils/helpers'
 
 interface Props {
   rorid: string
   isBot?: boolean
 }
 
-export default async function Content(props: Props) {
-  const { rorid, isBot = false } = props
-
-  const { data, error } = await fetchOrganization(rorid)
+export default function Content(props: Props) {
+  const { rorid } = props
+  const { data, error, loading } = useROROrganization(rorid)
+  if (loading) return <Loading />
+  const organization = data?.organization || {} as OrganizationType
 
-  if (error) return (
+  if (error || !organization) return (
     <Col md={{ span: 9, offset: 3 }}>
-      <Error title="An error occured." message={error.message} />
+      <Error title="An error fetching Organization." />
```

Comment:
Was this meant to say "An error occurred fetching Organization"?

Reply:
Will update to "An error occurred fetching the Organization".